service/controller/machinepoolexp/controller.go

```diff
@@ -0,0 +1,114 @@
+package machinepoolexp
```

Comment:
Existing MachinePool controller will now reconcile new non-experimental MachinePool CRs, so here we add a new MachinePool controller to reconcile old experimental CRs.

Its only job is to create new MachinePool and AzureMachinePool and set Metadata and Spec by copying it from old experimental MachinePool and AzureMachinePool.